api/src/main/java/org/openmrs/module/appointments/helper/AppointmentServiceHelper.java

```diff
@@ -0,0 +1,100 @@
+package org.openmrs.module.appointments.helper;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.openmrs.api.APIException;
+import org.openmrs.module.appointments.model.Appointment;
+import org.openmrs.module.appointments.model.AppointmentAudit;
+import org.openmrs.module.appointments.model.AppointmentStatus;
+import org.openmrs.module.appointments.validator.AppointmentStatusChangeValidator;
+import org.openmrs.module.appointments.validator.AppointmentValidator;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
```

Comment:
Is this really a component? To me, it looks like a simple helper class providing few static methods.
Are we mocking `AppointmentServiceHelper` in Unit tests?